content/references/protocol-reference/transactions/transaction-types/ammbid.md

```diff
@@ -0,0 +1,136 @@
+---
+html: ammbid.html
+parent: transaction-types.html
+blurb: Bid on an Automated Market Maker's auction slot, which grants a discounted fee.
+labels:
+  - AMM
+status: not_enabled
+---
+# AMMBid
+[[Source]](https://github.com/gregtatcam/rippled/blob/amm-core-functionality/src/ripple/app/tx/impl/AMMBid.cpp "Source")
+<!-- TODO: Update source link to merged version when available -->
+
+{% include '_snippets/amm-disclaimer.md' %}
+
+Bid on an Automated Market Maker's (AMM's) auction slot. If this transaction succeeds, you win the bid and can trade against the AMM at a discounted fee until you are outbid or 24 hours have passed. If you are outbid before 24 hours have passed, you are refunded part of the cost of your bid based on how much time remains.
```

Comment:
```suggestion
You win the bid and can trade against the AMM at a discounted fee until you are outbid or 24 hours have passed.
```
We can probably remove mentioning this is a transaction since we're already under transaction types.